Common/cpp/NativeModules/NativeReanimatedModule.cpp

```diff
@@ -369,10 +369,12 @@ jsi::Value NativeReanimatedModule::configureLayoutAnimation(
     jsi::Runtime &rt,
     const jsi::Value &viewTag,
     const jsi::Value &type,
+    const jsi::Value &sharedTransitionTag,
     const jsi::Value &config) {
   layoutAnimationsManager_.configureAnimation(
       viewTag.asNumber(),
       type.asString(rt).utf8(rt),
+      sharedTransitionTag.asString(rt).utf8(rt),
```

Comment:
Does this also apply to regular layout animations?

Reply:
Yes, but regular layout animation doesn't use this argument. I didn't want to add another JSI function to almost the same registration process, so i added additional argument for existing one.